authnz-theme/src/keycloak-theme/login/Template.tsx

```diff
@@ -130,6 +210,295 @@ export default function Template(props: TemplateProps<KcContext, I18n>) {
+									</DropdownMenuContent>
+								</DropdownMenu>
+
+								{/* TODO: unable to select option for some reason, unsure if from storybook */}
+								<Combobox
+									className="w-full md:max-w-xs"
+									options={localizationOptions}
+									initialValue={currentLocalizationOption}
+									selectPlaceholder="Select language" // TODO: resources
+									searchPlaceholder="Search language..." // TODO: resources
+									noResultsText="No language found" // TODO: resources
+									onChange={onChangeLocale}
+								/>
+							</div>
+						)}
+						<CardTitle className="flex flex-col gap-2">
+							{!displayRequiredFields && <PageHeader />}
+
+							{displayRequiredFields && (
+								<div
+									className={kcClsx("kcContentWrapperClass")}>
+									<div
+										className={clsx(
+											kcClsx("kcLabelWrapperClass"),
+											"subtitle",
+										)}>
+										<span className="subtitle">
+											<span className="required">*</span>
+											{msg("requiredFields")}
+										</span>
+									</div>
+									<div className="col-md-10">
+										<PageHeader />
+									</div>
+								</div>
+							)}
+
+							{displayMessage &&
+								message &&
+								(message.type !== "warning" ||
+									!isAppInitiatedAction) && (
+									<CardDescription className="flex gap-1 items-center">
+										<span className="hidden md:block">
+											{message.type === "success" && (
+												<CircleCheck className="h-4 w-4" />
+											)}
+											{message.type === "warning" && (
+												<CircleAlert className="h-4 w-4" />
+											)}
+											{message.type === "error" && (
+												<CircleX className="h-4 w-4" />
+											)}
+											{message.type === "info" && (
+												<Info className="h-4 w-4" />
+											)}
+										</span>
+
+										{toPlainText(message.summary)}
+									</CardDescription>
+								)}
+						</CardTitle>
+					</CardHeader>
+					<CardContent>{children}</CardContent>
+					<CardFooter className="flex-col gap-8">
+						{auth !== undefined && auth.showTryAnotherWayLink && (
+							<form
+								id="kc-select-try-another-way-form"
+								action={url.loginAction}
+								method="post">
+								<div className={kcClsx("kcFormGroupClass")}>
+									<div className={kcClsx("kcFormGroupClass")}>
+										<input
+											type="hidden"
+											name="tryAnotherWay"
+											value="on"
+										/>
+										<a
+											href="#"
+											id="try-another-way"
+											onClick={() => {
+												document.forms[
+													"kc-select-try-another-way-form" as never
+												].submit();
+												return false;
+											}}>
+											{msg("doTryAnotherWay")}
+										</a>
+									</div>
+								</div>
+							</form>
+						)}
+						<div className="grid grid-flow-row grid-rows-2 md:grid-cols-3 gap-4">
+							{socialProviders.map(socialProvider => {
+								return (
+									<Button
```

Comment:
TODO: update svgs to "currentColor" and also in light mode is outline and otherwise primary:

<img width="1082" alt="image" src="https://github.com/user-attachments/assets/7141e4aa-3f43-49f4-b015-bbcf492bc05a">

Reply:
<img width="1082" alt="image" src="https://github.com/user-attachments/assets/c899cc28-d3d8-492c-8460-e71516f3cb19">

Reply:
<img width="1082" alt="image" src="https://github.com/user-attachments/assets/fe00e4e7-5579-45b5-8e74-d34380095efa">